.github/workflows/ci.yml

```diff
@@ -3,6 +3,19 @@ name: CI
 on: [push, pull_request]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip
```

Comment:
I don't think this line is necessary? `actions/setup-python` already gives us the latest version (see output from this on your branch)

Reply:
Not always...  Watch it over time.